scalameta/common/shared/src/main/scala/scala/meta/internal/trees/registry.scala

```diff
@@ -29,7 +29,7 @@ class RegistryMacros(val c: Context) extends AstReflection with MacroHelpers {
       ) = mdef
       val enclosingUnit = c.asInstanceOf[{ def enclosingUnit: { def body: Tree } }].enclosingUnit
         .body
-      val anns1 = anns :+ q"new $AstMetadataModule.registry(${enclosingUnit.detectAst})"
+      val anns1 = anns :+ q"new $AstMetadataModule.registry(..${enclosingUnit.detectAst})"
```

Comment:
Apologies, this and the other changes to scala 2 trees (outside of the `Quasi` one), were leftovers from attempts to write the codegen part in Scala 3 (instead of how it ended up now, in Scala 2) that somehow got committed.